plugins/optimization-detective/tests/storage/test-data.php

```diff
@@ -498,6 +519,21 @@ static function ( $link, $author_id ) {
 	 * @covers ::od_get_current_theme_template
 	 */
 	public function test_od_get_current_url_metrics_etag( Closure $set_up ): void {
+		update_option(
+			'active_plugins',
+			array(
+				'optimization-detective/load.php',
+			)
+		);
+		if ( is_multisite() ) {
+			update_site_option(
+				'active_sitewide_plugins',
+				array(
+					'performance-lab/load.php',
+				)
+			);
```

Comment:
See above, this test is not using the data how it should actually be provided:
```suggestion
			update_site_option(
				'active_sitewide_plugins',
				array(
					'performance-lab/load.php' => time(),
				)
			);
```